packages/grafana-e2e/src/flows/importDashboard.ts

```diff
@@ -0,0 +1,55 @@
+import { DeleteDashboardConfig } from '.';
+import { e2e } from '../index';
+import { fromBaseUrl, getDashboardUid } from '../support/url';
+
+type Panel = {
+  title: string;
+  [key: string]: any;
+};
+
+type Dashboard = { title: string; panels: Panel[]; [key: string]: any };
+
+/**
+ * Smoke test a datasource by quickly importing a test dashboard for it
+ * @param dashboardToImport a sample dashboard
+ */
+export const importDashboard = (dashboardToImport: Dashboard) => {
+  e2e().visit(fromBaseUrl('/dashboard/import'));
+
+  // Note: normally we'd use 'click' and then 'type' here, but the json object is so big that using 'val' is much faster
+  e2e.components.DashboardImportPage.textarea().click({ force: true }).invoke('val', JSON.stringify(dashboardToImport));
+  e2e.components.DashboardImportPage.submit().click({ force: true });
+  e2e.components.ImportDashboardForm.name().click({ force: true }).clear().type(dashboardToImport.title);
+  e2e.components.ImportDashboardForm.submit().click({ force: true });
+  e2e().wait(3000);
+
+  // save the newly imported dashboard to context so it'll get properly deleted later
+  e2e()
+    .url()
+    .should('contain', '/d/')
+    .then((url: string) => {
+      const uid = getDashboardUid(url);
+
+      e2e.getScenarioContext().then(({ addedDashboards }: any) => {
+        e2e.setScenarioContext({
+          addedDashboards: [...addedDashboards, { title: dashboardToImport.title, uid } as DeleteDashboardConfig],
+        });
+      });
+    });
```

Comment:
Rather than using `any` here, would it be possible to give it the correct type (and then the casting wouldn't be needed later)?